evm/src/helpers.ts

```diff
@@ -80,7 +90,7 @@ export const linkContract = async (account: any): Promise<any> => {
   const receipt = await web3.eth.sendTransaction({
     data: linkToken.bytecode,
     from: account,
-    gasLimit: 2000000,
+    gas: 2000000,
```

Comment:
Was this having no effect, or what changed?

Reply:
`@types/web3` doesn't have `gasLimit` as a alias, since i believe its an unofficial property that was introduced by metamask(AFAIK, dont remember who introduced it).